proposals/lip-0043.md

```diff
@@ -32,7 +32,7 @@ In this LIP, the registration process introduced in the previous section is spec
 
 ### Sidechain Registration Process
 
-As mentioned above, for a sidechain to be interoperable in the Lisk ecosystem, it has to be registered in the Lisk Mainchain via a sidechain registration command. A transaction with this command can be sent by any user account in the Lisk Mainchain with enough funds to pay the required fee. The processing of this command implies the creation of a _sidechain account_ in the mainchain state associated with a unique chain identifier and a name. This means that every new sidechain occupies a certain namespace in the ecosystem. Additionally, every newly registered sidechain can increase the size of every cross-chain update command posted on the mainchain (due to the increasing size of the [`outboxRootWitness`][lip-0053#outbox-root-witness] property of the command). For these two reasons, the minimum fee for this command has an added constant similar to the extra fee in a [delegate registration command][docs:delegate]. The value of this extra registration fee is `REGISTRATION_FEE` LSK.
+As mentioned above, for a sidechain to be interoperable in the Lisk ecosystem, it has to be registered in the Lisk Mainchain via a sidechain registration command. A transaction with this command can be sent by any user account in the Lisk Mainchain with enough funds to pay the required fee. The processing of this command implies the creation of a _sidechain account_ in the mainchain state associated with a unique chain identifier and a name. This means that every new sidechain occupies a certain namespace in the ecosystem. Additionally, every newly registered sidechain can increase the size of every mainchain cross-chain update command posted on the sidechain (due to the increasing size of the [`outboxRootWitness`][lip-0053#outbox-root-witness] property of the command). For these two reasons, the minimum fee for this command has an added constant similar to the extra fee in a validator registration command. The value of this extra registration fee is `REGISTRATION_FEE` LSK.
```

Comment:
"of every mainchain cross-chain update command posted on the sidechain" --> " ... posted on sidechains"